bucket/glitter.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.0.0",
+    "description": "Git tooling of the future",
+    "homepage": "https://github.com/Milo123459/glitter",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Milo123459/glitter/releases/download/v1.0.0/glitter-x86_64-pc-windows-msvc.zip",
+            "hash": "83a8bfa0e7ec2cbde05c27c51e6ccff1ce376db2"
```

Comment:
These should be the checksum of the file itself, not the commit sha1.

Reply:
alright